examples/pytorch/stgcn_wave/main.py

```diff
@@ -0,0 +1,125 @@
+import dgl
+import random
+import torch
+import numpy as np
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+from load_data import *
+from utils import *
+from model import *
+from sensors2graph import *
+import torch.nn as nn
+import argparse
+import scipy.sparse as sp
+
+parser = argparse.ArgumentParser(description='STGCN_WAVE')
+parser.add_argument('--lr', default=0.001, type=float, help='learning rate')
+parser.add_argument('--disablecuda', action='store_true', help='Disable CUDA')
+parser.add_argument('--batch_size', type=int, default=50, help='batch size for training and validation (default: 50)')
+parser.add_argument('--epochs', type=int, default=50, help='epochs for training  (default: 50)')
+parser.add_argument('--num_layers', type=int, default=9, help='number of layers')
+parser.add_argument('--window', type=int, default=144, help='window length')
+parser.add_argument('--sensorsfilepath', type=str, default='./data/sensor_graph/graph_sensor_ids.txt', help='sensors file path')
+parser.add_argument('--disfilepath', type=str, default='./data/sensor_graph/distances_la_2012.csv', help='distance file path')
+parser.add_argument('--tsfilepath', type=str, default='./data/metr-la.h5', help='ts file path')
+parser.add_argument('--savemodelpath', type=str, default='./save/stgcnwavemodel.pt', help='save model path')
+parser.add_argument('--pred_len', type=int, default=5, help='how many steps away we want to predict')
+parser.add_argument('--control_str', type=str, default='TNTSTNTST', help='model strcture controller, T: Temporal Layer, S: Spatio Layer, N: Norm Layer')
+parser.add_argument('--channels', type=list, default=[1, 16, 32, 64, 32, 128], help='model strcture controller, T: Temporal Layer, S: Spatio Layer, N: Norm Layer')

```

Comment:
It seems that using `type=list` in `argparse` is a [bad idea](https://stackoverflow.com/questions/15753701/how-can-i-pass-a-list-as-a-command-line-argument-with-argparse).  Did you test it?  How should I supply a list of integers in command line?